src/components/SelectedSkills/styles.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+const SelectedSkillsWrapper = styled.div`
+  align-items: center;
+  margin: 20px 0;
+  border: 1px solid white;
+  border-radius: 5px;
+  padding: 10px 0 10px 10px;
+  /* visibility: hidden; */
```

Comment:
Do we really need this uncommented code?

Reply:
Correction done in Phase-2 branch.